service_quotas/service_quotas.go

```diff
@@ -101,13 +110,14 @@ func NewServiceQuotas(region, profile string) (QuotasInterface, error) {
 	}
 
 	quotasService := awsservicequotas.New(awsSession, aws.NewConfig().WithRegion(region))
-	checks := newUsageChecks(awsSession, aws.NewConfig().WithRegion(region))
+	checks, availabilityChecks := newUsageChecks(awsSession, aws.NewConfig().WithRegion(region))
```

Comment:
Maybe it makes sense to rename that to something like `serviceQuotasChecks` and `otherChecks`?